projects/testing-library/src/lib/models.ts

```diff
@@ -186,6 +186,21 @@ export interface RenderComponentOptions<ComponentType, Q extends Queries = typeo
    * })
    */
   componentProviders?: any[];
+  /**
+   * @description
+   * A collection of imports to override a standalone component's imports with.
+   *
+   * @default
+   * undefined
```

Comment:
To be consistent with the other default properties, I usually use an empty array.

```suggestion
   * []
```